port/raspberrypi/rp2xxx/src/hal/pio/rp2040.zig

```diff
@@ -0,0 +1,112 @@
+const std = @import("std");
+const assert = std.debug.assert;
+
+const microzig = @import("microzig");
+
+const common = @import("common.zig");
+
+const gpio = @import("../gpio.zig");
+const resets = @import("../resets.zig");
+const hw = @import("../hw.zig");
+
+pub const Pio = enum(u1) {
+    pio0 = 0,
+    pio1 = 1,
+
+    pub fn reset(self: Pio) void {
```

Comment:
Not a bug you introduced but go ahead and delete this function for now. It won't compile as is if you actually tried to use this function anways. For context this is the _hard_ reset bit for a given peripheral at the processor level and generally there's no real reason you want to do this. If someone really wants to do it they can use the `resets.zig` module! Will comment on the rp2350 implementation but same deal.